File: insertion.php
<?php
include ("connection.php");

if(isset($_POST['submit'])){
    $sql="INSERT INTO `layer_details`(`id`, `name`, `parent_id`) VALUES ('','".$_REQUEST['name']."','".$_REQUEST['level']."')";
    $result=mysqli_query($con,$sql);
    if($result){
        echo "<h1>"."data inserted";
    }
}

?>

File: insertion.py
import mysql.connector as connector
con=connector.connect(host="localhost",port='3306',user='root',
                      password='',database='test')

#query = "create table shreya(id int,name char(244))"

def insert_data(connection,table,data):
        if(table!=''):
            cur = connection.cursor()
            sql = "INSERT INTO "+table+" VALUES (%s,%s,%s)"
            #sql1 = "insert into student('id','Namel','Course') values(1,'ABC','ABC')"
            #cur.execute(sql1)
            cur.execute(sql,data)
            connection.commit()

          
        else:
            print("Data not submitted")

  
insert_data(con,'student',['6','ABC','ABC'])



